doc/users/next_whats_new/widget_horizontal.rst

```diff
@@ -0,0 +1,23 @@
+RadioButtons widget may now be laid out horizontally
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The `.RadioButtons` widget's primary layout direction may now be specified with
+the *orientation* keyword argument:
```

Comment:
Is *orientation* the right name? I know we use it in some places, but more in the sense of rotation than arrangement.

Possible alternatives: arrangement, direction, layout, ...

Reply:
Boxplot is a counter example. There (as well as bar) it's associated with the rotation of the box = direction of the whiskers. "Vertical" means the distribution is along the y axis and multiple boxes are arranged side-by-side horizontally.

In contrast, here "vertical" would mean multiple radio buttons are arranged stacked vertically.

Reply:
Yeah, I'd say "pack_direction" or "layout_direction".  This isn't going to be a heavily used kwarg, so we can be verbose.

Reply:
"layout_direction" sounds good.

Reply:
With respect to https://github.com/matplotlib/matplotlib/pull/27946#issuecomment-2123384239, is that something we want to use `orientation` for (giving that this change would be moving toward `layout_direction`)?

Reply:
Do you mean text above or to the side of the button? I would not make this configurable but always put the text to the side. This is marginally more work, but the standard look and feel of radio buttons. If we want to add the horizontal layout, IMHO we have to do it properly.